cmake/Package.cmake

```
@@ -0,0 +1,44 @@
+set(CPACK_PACKAGE_VENDOR "khronos")
+
+set(CPACK_PACKAGE_DESCRIPTION "OpenCL API C headers")
+
+set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/LICENSE")
+
+set(CPACK_RESOURCE_FILE_README "${CMAKE_CURRENT_SOURCE_DIR}/README.md")
+
+if(NOT CPACK_PACKAGING_INSTALL_PREFIX)
+  set(CPACK_PACKAGING_INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}")
+endif()
+
+if(NOT CPACK_SOURCE_IGNORE_FILES)
+  set(CPACK_SOURCE_IGNORE_FILES="/.github/;/build/")
+endif()
+
+# DEB packaging configuration
+set(CPACK_DEBIAN_PACKAGE_MAINTAINER ${CPACK_PACKAGE_VENDOR})
+
+set(CPACK_DEBIAN_PACKAGE_HOMEPAGE
+    "https://github.com/KhronosGroup/OpenCL-Headers")
+
+# Get Debian package name from project name
+string(TOLOWER ${PROJECT_NAME} DEBIAN_PACKAGE_NAME)
+string(REPLACE "opencl" "opencl-" DEBIAN_PACKAGE_NAME ${DEBIAN_PACKAGE_NAME})
```

There's no need in being smart about it. If the project name changes, this fragment will very likely have to change too. Then just hardcode the result.
```suggestion
set(DEBIAN_PACKAGE_NAME "opencl-headers")
```

---

Done!